### PR TITLE
🔐 Use HTTP2 with Web Test Runner

### PR DESCRIPTION
**Pull request by Sakchai-Refinitiv**
## Description

🔐 Enable HTTP2 in Web Test Runner to improve performance and stability when running on BrowserStack or Playwright.

## Type of change

- [ ] Refactor (improves code without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: web-test-runner.config.mjs [hunk 1]
```diff
@@ -37,16 +37,30 @@ export default {
...
-      headless: true,
-      args: [
-        '--disable-setuid-sandbox',
-        '--disable-extensions'
-      ]
+    playwrightLauncher({
+      product: 'chromium',
+      createBrowserContext: ({browser}) => browser.newContext({ignoreHTTPSErrors: true}),
+      launchOptions: {
+        headless: true,
```
Comment (bualoy-napat) [inline web-test-runner.config.mjs]: I am not sure that do we really need to set `headless` to true ? . I just went to the type description of `headless` property and they said that `headless` was set to `true` by default.

Comment (Sakchai-Refinitiv) [reply web-test-runner.config.mjs]: Yes, we can remove it for clean code. 😀

Comment (Sakchai-Refinitiv) [reply web-test-runner.config.mjs]: removed

**Review by bualoy-napat — APPROVED** [review]
This look good to me

**Pull request merged**